docker-compose-rag-doll-only.yml

```diff
@@ -3,15 +3,6 @@ services:
     build: assistant
     environment:
       ASSISTANT_LANGUAGES: ${ASSISTANT_LANGUAGES}
-      SYSTEM_PROMPT_en: ${SYSTEM_PROMPT_en}
```

Comment:
Please also make a pull request on `akvo-config` to clean up the now-unused prompt files there.

Reply:
This is still pending.

Reply:
I have created a PR on `akvo-config` and request your review @kjkoster